### ISSUE TITLE
 Could not find com.eightbitlab:blurview:1.6.3.

### ISSUE THREAD

**Issue by vinaysgl** [Kureev/react-native-blur#446]
Not able to run the application getting the error

       
      `
Execution failed for task ':app:checkDebugAarMetadata'.
> Could not resolve all files for configuration ':app:debugRuntimeClasspath'.
   > Could not find com.eightbitlab:blurview:1.6.3.
     Searched in the following locations:
       - https://dl.google.com/dl/android/maven2/com/eightbitlab/blurview/1.6.3/blurview-1.6.3.pom
       - https://repo.maven.apache.org/maven2/com/eightbitlab/blurview/1.6.3/blurview-1.6.3.pom
       - file:/Users/vin/.m2/repository/com/eightbitlab/blurview/1.6.3/blurview-1.6.3.pom
       - file:/Users/vin/Desktop/unikapp/node_modules/react-native/android/com/eightbitlab/blurview/1.6.3/blurvie
      `

**Comment by chungweileong94** [comment]
Or technically, from the time being, we can still keep the `jcenter` in `/android/app/build.gradle` until a proper fix released, the reason why RN 0.65 remove it is because [jcenter repo will be read-only](https://developer.android.com/studio/build/jcenter-migration)

```diff
allprojects {
    repositories {
        mavenCentral()
        mavenLocal()
        maven {
            // All of React Native (JS, Obj-C sources, Android binaries) is installed from npm
            url("$rootDir/../node_modules/react-native/android")
        }
        maven {
            // Android JSC is installed from npm
            url("$rootDir/../node_modules/jsc-android/dist")
        }

        google()
+       jcenter()
        maven { url 'https://www.jitpack.io' }
    }
}
```

**Comment by oyelaappdev** [comment]
> ```diff
>   jcenter()
> ```

android/build.gradle

**Comment by Dror-Bar** [comment]
When I have both mavenCentral() and jcenter I get an error:
Corrupt serialized resolution result. Cannot find selected module (116) for constraint debugCompileClasspath -> com.facebook.android:facebook-android-sdk:{strictly 9.0.0}
while the linked solution (patch-package) didn't work for me

**Comment by tomskopek** [comment]
Looks like the [latest commit three days ago](https://github.com/Kureev/react-native-blur/commit/6231655f961637701adc4672be2eed08e4cacef1) addresses this, but a release has not been made (as of May 12, 2022).
I got past this by adding the package in my package.json like so, which references this commit:
```json
  "dependencies": {
    ...,
    "@react-native-community/blur": "Kureev/react-native-blur#6231655f961637701adc4672be2eed08e4cacef1",
    ...
    }
```

**Comment by idrissakhi** [comment]
i have re-written the lib to work correctly on new RN version => https://www.npmjs.com/package/rn-id-blurview
need some support so the react community take it to replace this package that seems to not be maintained anymore

**Comment by saifbak** [comment]
Apparently, this worked for me. you have to put `implementation 'com.eightbitlab:blurview:1.6.3'` in android/app/build.gradle
 and then PUT jcenter(). and it has been successfully created a build.


```
allprojects {
    repositories {
        mavenCentral()
        mavenLocal()
        maven {
            // All of React Native (JS, Obj-C sources, Android binaries) is installed from npm
            url("$rootDir/../node_modules/react-native/android")
        }
        maven {
            // Android JSC is installed from npm
            url("$rootDir/../node_modules/jsc-android/dist")
        }

        google()
         jcenter()
        maven { url 'https://www.jitpack.io' }
    }
}
```

**Comment by vladimirevstratov** [comment]
> Or technically, from the time being, we can still keep the `jcenter` in `/android/app/build.gradle` until a proper fix released, the reason why RN 0.65 remove it is because [jcenter repo will be read-only](https://developer.android.com/studio/build/jcenter-migration)
> 
> ```diff
> allprojects {
>     repositories {
>         mavenCentral()
>         mavenLocal()
>         maven {
>             // All of React Native (JS, Obj-C sources, Android binaries) is installed from npm
>             url("$rootDir/../node_modules/react-native/android")
>         }
>         maven {
>             // Android JSC is installed from npm
>             url("$rootDir/../node_modules/jsc-android/dist")
>         }
> 
>         google()
> +       jcenter()
>         maven { url 'https://www.jitpack.io' }
>     }
> }
> ```

thx, it works

**Comment by charlyBerthet** [comment]
This works as well
```
allprojects {
    repositories {
        //...
        jcenter() {
            content {
                includeModule("com.eightbitlab", "blurview")
            }
        }
    }
}
```

**Comment by Titozzz** [comment]
fixed in 4.0.0

**Comment by chenop** [comment]
jcenter is deprecated so it is better not to include it in build.gradle.
BUT - some libraries still can be fetched only from jcenter.
So to specifically fetch those libraries - you can add:
`     jcenter() {
            content {
                // Other libraries that exist only in jcenter
                includeModule("com.eightbitlab", "blurview")
            }
        }
`
